client/eventsSDL/InputHandler.cpp

```diff
@@ -217,6 +217,18 @@ void InputHandler::preprocessEvent(const SDL_Event & ev)
 #endif
 		return;
 	}
+	else if(ev.type == SDL_APP_WILLENTERBACKGROUND)
+	{
+		std::scoped_lock interfaceLock(ENGINE->interfaceMutex);
+		ENGINE->user().onAppPaused(false);
+		return;
+	}
+	else if(ev.type == SDL_APP_WILLENTERFOREGROUND)
+	{
+		std::scoped_lock interfaceLock(ENGINE->interfaceMutex);
+		ENGINE->user().onAppPaused(true);
```

Comment:
This will trigger second autosave on entering *foreground*. Am I right? Probably unnecessary.

Reply:
Yes. Forgot if for `resume` flag. Would like to keep `SDL_APP_WILLENTERFOREGROUND` as we can use resume flag for later improvements. And it's imo a little bit more understandable.